Maya/Exporter/BabylonExporter.GLTFExporter.Material.cs

```diff
@@ -562,47 +562,69 @@ private bool isTextureOk(BabylonTexture texture)
         BabylonColor3 dielectricSpecular = new BabylonColor3(0.04f, 0.04f, 0.04f);
         const float epsilon = 1e-6f;
 
-        private MetallicRoughness ConvertToMetallicRoughness(SpecularGlossiness specularGlossiness, bool displayPrints = false)
-        {
-            var diffuse = specularGlossiness.diffuse;
-            var opacity = specularGlossiness.opacity;
-            var specular = specularGlossiness.specular;
-            var glossiness = specularGlossiness.glossiness;
-
-            var oneMinusSpecularStrength = 1 - specular.getMaxComponent();
-            var metallic = solveMetallic(diffuse.getPerceivedBrightness(), specular.getPerceivedBrightness(), oneMinusSpecularStrength);
-
-            var diffuseScaleFactor = oneMinusSpecularStrength / (1 - dielectricSpecular.r) / Math.Max(1 - metallic, epsilon);
-            var baseColorFromDiffuse = diffuse.scale(diffuseScaleFactor);
-            var baseColorFromSpecular = specular.subtract(dielectricSpecular.scale(1 - metallic)).scale(1 / Math.Max(metallic, epsilon));
-            var baseColor = BabylonColor3.Lerp(baseColorFromDiffuse, baseColorFromSpecular, metallic * metallic).clamp();
-            //var baseColor = baseColorFromDiffuse.clamp();
-
-            if (displayPrints)
-            {
-                RaiseVerbose("-----------------------", 3);
-                RaiseVerbose("diffuse=" + diffuse, 3);
-                RaiseVerbose("opacity=" + opacity, 3);
-                RaiseVerbose("specular=" + specular, 3);
-                RaiseVerbose("glossiness=" + glossiness, 3);
-
-                RaiseVerbose("oneMinusSpecularStrength=" + oneMinusSpecularStrength, 3);
-                RaiseVerbose("metallic=" + metallic, 3);
-                RaiseVerbose("diffuseScaleFactor=" + diffuseScaleFactor, 3);
-                RaiseVerbose("baseColorFromDiffuse=" + baseColorFromDiffuse, 3);
-                RaiseVerbose("baseColorFromSpecular=" + baseColorFromSpecular, 3);
-                RaiseVerbose("metallic * metallic=" + metallic * metallic, 3);
-                RaiseVerbose("baseColor=" + baseColor, 3);
-                RaiseVerbose("-----------------------", 3);
-            }
-
-            return new MetallicRoughness
-            {
-                baseColor = baseColor,
-                opacity = opacity,
-                metallic = metallic,
-                roughness = 1 - glossiness
-            };
+        private MetallicRoughness ConvertToMetallicRoughness(SpecularGlossiness specularGlossiness, bool displayPrints = false)
+        {
+            // Hard coded points used to define the specular power to roughness curve.

```

Comment:
Are these points supposed to be identical to the glTF serializer in Babylon.js? Are the units for specular power the same between the standard material for Babylon and Max?

Reply:
yes, but not really?

here's two test renders I made using a directional light  and spheres of varying SpecularPower(Babylon)/glossiness(3dsMax) using the same values:

Babylon:
![specspectum_babylon (3)](https://user-images.githubusercontent.com/3952729/56535791-654e7200-6511-11e9-919d-46846e9b7ec1.png)

3dsMax:
![specspectum_3dsmax](https://user-images.githubusercontent.com/3952729/56535832-7e572300-6511-11e9-9429-ebc53b97b779.png)

Reply:
It gets even worse when exporting from Max to Babylon, we get this:
![screenshot_19-4-22_17-22](https://user-images.githubusercontent.com/3952729/56542453-625c7d00-6523-11e9-9d62-f31050980d98.png)


I'm willing to accept the disparity between different implementaitons of the Blinn lighting model as an unsolved mystery.

Reply:
as we discussed offline, this is caused by the 3dsMax->Babylon portion of the converter, I'm not quite sure if there is a linear relationship between 3dsMax and BabylonJS's shininess units.

But, I believe for the sake of "correctness" we can assume in our Babylon->GLTF part of the conversion, we are only working in Babylon units.